string-manipulation-3/src/reverse-words.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable no-unused-vars */
+function reverseWords(string) {
+  var stringArray = string.split(' ');
+  var reverseString = '';
+  for (var i = 0; i < stringArray.length; i++) {
+    var current = stringArray[i];
+    for (var j = current.length - 1; j >= 0; j--) {
+      reverseString += current[j];
+      if (i !== stringArray.length - 1) {
+        if (j === 0) {
```

Comment:
instead of nesting `if` statements with no `else`, try to stick with `&&` in one `if` statement.